packages/tablesorter/src/providers/JSONDataProvider.ts

```diff
@@ -34,7 +34,7 @@ import * as d3 from "d3";
 
 const debug = require("debug"); // tslint:disable-line
 const log = debug("essex:widget:tablesorter:JSONDataProvider");
-import ldIsNaN = require("lodash/isNaN");
+const ldIsNaN = require("lodash/lang/isNaN"); //tslint:disable-line
```

Comment:
My plan was to keep it the same as the line I removed, `import ldIsNan = require..` which works great.  But, I downgraded to `3.6.0`, cause that is what the app has been tested against, and it seems that the `@types/lodash` either doesn't support lodash 3, or they weren't defining the individual functions as modules until lodash 4.  Either way, typescript wasn't picking up the typings using the `import` style.